invoicing/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from enumfields import EnumField
+
+from invoicing.enums import (
+    InstallmentPercentageSpecifier,
+    InstallmentType,
+    InstallmentUnit,
+)
+
+
+class InstallmentBase(models.Model):
+    created_at = models.DateTimeField(verbose_name=_("created at"), auto_now_add=True)
```

Comment:
I recommend to inherit from TimestampedModel instead: https://github.com/City-of-Helsinki/apartment-application-service/blob/develop/apartment_application_service/models.py#L4

Reply:
The idea is that these installment objects will be regenerated every time they are updated, so there is no need for the `update_at` field.